_config/forms.yml

```
@@ -1,6 +1,10 @@
 ---
 Name: adminforms
 ---
+SilverStripe\Forms\GridField\GridFieldDetailForm_ItemRequest:
+  extensions:
+    - SilverStripe\Admin\GridFieldDetailFormPreviewExtension
```

```suggestion
    - SilverStripe\Admin\Forms\GridFieldDetailFormPreviewExtension
```

See comment futher down about updating namespace